cosmos/migrations/0002_remove_profile_gender.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.0.8 on 2020-07-22 10:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cosmos", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="profile", name="gender",),
+    ]
```

Comment:
squashed on my local branch, but I was informed that I shouldn't be deleting the previous migrations unless all instances have been migrated. did any of you `python manage.py migrate` on this branch lately? If not, I can safely delete the redundant migrations
![image](https://user-images.githubusercontent.com/19555626/88263823-f4c17280-ccca-11ea-91b2-d3fe3778ecea.png)
@bcvandendool @Marti2203

Reply:
yeah should be fine, otherwise ill just drop the database i have locally. its fine to have to do that kinda stuff when it still is a pull request imho